CycloneDX/cdxgen
# Reachable slices + Bug fixes

- [ ] Support for generating call stack evidence based on reachables (aka evidence of reachable components)
- [ ] Improved yarn.lock v1 parsing to deal with aliases
- [ ] Fixes #653 #649 
- [ ] Improves support for docker library images by retrying without the library prefix
- [ ] Reliably disable any db updates from trivy by creating a fake cache dir and excluding jar files. Workaround for https://github.com/CycloneDX/cdxgen-plugins-bin/issues/9

## How to test?

See the delta in repotests.

1. cdxgen must be invoked with the `--deep` argument, and bom.json must be placed within the same application directory
2. run evinse using the `--with-reachables` argument

https://github.com/CycloneDX/cdxgen/blob/feature/reachable-slice/.github/workflows/repotests.yml#L158-L159
